antismash/common/secmet/features/feature.py

```diff
@@ -201,9 +208,14 @@ def to_biopython(self, qualifiers: Dict[str, Any] = None) -> List[SeqFeature]:
             # adjust location back if neccessary
             if self._original_codon_start != 0:
                 feature.location = _adjust_location_by_offset(feature.location, -start)
+        if self._pseudo:
+            quals["pseudo"] = []
```

Comment:
Why not just set the value to `None` straight away? It removes the need for a special case 3 lines later.